uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -95,13 +95,7 @@
 import org.jasig.portal.portlets.fragmentadmin.FragmentAdministrationHelper;
 import org.jasig.portal.portlets.groupselector.EntityEnum;
 import org.jasig.portal.portlets.portletadmin.xmlsupport.IChannelPublishingDefinitionDao;
-import org.jasig.portal.security.AuthorizationPrincipalHelper;
-import org.jasig.portal.security.IAuthorizationPrincipal;
-import org.jasig.portal.security.IAuthorizationService;
-import org.jasig.portal.security.IPermission;
-import org.jasig.portal.security.IPermissionManager;
-import org.jasig.portal.security.IPerson;
-import org.jasig.portal.security.PermissionHelper;
+import org.jasig.portal.security.*;
```

Comment:
Actually it has.  There was discussion of adopting the Google Code standards.  One thing IMHO we might not is the actual order of the imports (Google Code standards have a different order than most of our files), but the Google Code standards do suggest never importing *.